src/rpc/blockchain.cpp

```diff
@@ -1094,15 +1092,12 @@ static UniValue verifychain(const JSONRPCRequest& request)
                 },
             }.Check(request);
 
-    LOCK(cs_main);
+    const int check_level(request.params[0].isNull() ? DEFAULT_CHECKLEVEL : request.params[0].get_int());
```

Comment:
nit: should also use list initialization here (`{`....`}`), same as one line below

Reply:
That wouldn't compile

Reply:
Oh I see, `DEFAULT_CHECKLEVEL` has the wrong type -- unsigned even If the consuming function takes it as signed (for whatever reason).